index.html

```diff
@@ -27,15 +34,8 @@
   except this little library grooves the <a
   href="https://github.com/facebook/react">React way</a>.</p>
 
-  <h2>Example</h2>
-
-  <iframe
-    width="100%"
-    height="300"
-    src="http://jsfiddle.net/L4z5wcx0/7/embedded/result,js,html,css/"
-    allowfullscreen="allowfullscreen"
-    frameborder="0">
-  </iframe>
+  <h2>Examples</h2>
+    <div id="react-waypoint-examples"></div>
```

Comment:
Now that we have two examples, that makes sense.
We could model the navigation after the 'Waypoints' documentation page:
Top Nav > Guides > Lists examples with left sub-nav to jump between them.
http://imakewebthings.com/waypoints/guides/getting-started/

Let me know what you think.